nimare/utils.py

```diff
@@ -103,6 +103,9 @@ def dict_to_coordinates(data, masker, space):
             con_df = pd.DataFrame(temp_data.T, columns=exp_columns)
             all_dfs.append(con_df)
 
+    if not all_dfs:
+        return pd.DataFrame({"id": []})
```

Comment:
Could you include these columns as well: "study_id", "contrast_id", "x", "y", "z", and "space".